airflow/www/utils.py

```diff
@@ -76,6 +76,97 @@ def is_accessible(self):
         )
 
 
+def generate_pages(current_page, num_of_pages,
+                   search=None, showPaused=None, window=7):
```

Comment:
docstring would be nice, plus explanation of window=7

Reply:
added, thanks.